plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/Scope.h

```diff
@@ -0,0 +1,138 @@
+
+#if MAYA_API_VERSION >= 20190200 && MAYA_API_VERSION < 20200000
+class Scope
+  : public MPxTransform_BoundingBox,
+    public AL::maya::utils::NodeHelper
+#else
+class Scope
+  : public MPxTransform,
+    public AL::maya::utils::NodeHelper
+#endif
+{
+public:
+
+  Scope();
+  virtual ~Scope();
+
+  //--------------------------------------------------------------------------------------------------------------------
+  // Type Info & Registration
+  //--------------------------------------------------------------------------------------------------------------------
+  AL_MAYA_DECLARE_NODE();
+
+  //--------------------------------------------------------------------------------------------------------------------
+  // Input Attributes
+  //--------------------------------------------------------------------------------------------------------------------
+  AL_DECL_ATTRIBUTE(primPath);
+  AL_DECL_ATTRIBUTE(inStageData);
+
+  //--------------------------------------------------------------------------------------------------------------------
+  /// \name Methods
+  //--------------------------------------------------------------------------------------------------------------------
+
+  /// \brief  returns the transformation matrix for this transform node
+  /// \return the transformation matrix
+  virtual BasicTransformationMatrix* transform() const
+    { return reinterpret_cast<BasicTransformationMatrix*>(transformationMatrixPtr()); }
+
+  virtual const MObject getProxyShape() const
+    { return proxyShapeHandle.object(); }
+
+protected:
+
+  //--------------------------------------------------------------------------------------------------------------------
+  /// virtual overrides
+  //--------------------------------------------------------------------------------------------------------------------
+  MStatus compute(const MPlug &plug, MDataBlock &datablock) override;
+
+private:
+  MPxNode::SchedulingType schedulingType() const override
+    { return kParallel; }
+
+  MStatus validateAndSetValue(const MPlug& plug, const MDataHandle& handle, const MDGContext& context) override;
+  MPxTransformationMatrix* createTransformationMatrix() override;
+  void postConstructor() override;
+  MBoundingBox boundingBox() const override;
+  MStatus connectionMade(const MPlug& plug, const MPlug& otherPlug, bool asSrc) override;
+  MStatus connectionBroken(const MPlug& plug, const MPlug& otherPlug, bool asSrc) override;
+  bool isBounded() const override
+    { return true; }
+  bool treatAsTransform() const override
+    { return false; }
+
+  //--------------------------------------------------------------------------------------------------------------------
+  /// Data members
+  //--------------------------------------------------------------------------------------------------------------------
+  bool updateTransformInProgress = false;
```

Comment:
/git/usd/maya-usd/plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/Scope.h:110:8: error: private field 'updateTransformInProgress' is not used [-Werror,-Wunused-private-field]
  bool updateTransformInProgress = false;